Week2/part1.js

```diff
@@ -0,0 +1,81 @@
+var mysql = require('mysql');
+var connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'new_world',
+});
+
+connection.connect();
+
+function sendQuery(query) {
+  connection.query(query, function(error, results, fields) {
+    if (error) {
+      throw error;
+    }
+    if (!results[0]) {
+      console.log('Sorry no match in our database!');
+    }
+    if (results.length === 0) {
+      console.log('False');
+    } else {
+      console.log('The reply is', results);
+    }
+  });
+}
+
+const inputType = process.argv[2]; // type like region, language, country
+const userInput = process.argv[3]; // value like western europe, Dutch, Netherlands
+const userInput2 = process.argv[4];
+
+//What is the capital of country X ? (Accept X from user)
+const capital_of_country = `SELECT capital FROM country WHERE country.Name = '${userInput}'`;
```

Comment:
What would happen if the user with the query
`Spain; drop table country` ?
To protect your app against such attacks you need to use prepared statements. See an example here https://github.com/mysqljs/mysql#preparing-queries

Reply:
It's done! Thanks for the information! I appreciate it! :)